postreise/analyze/helpers.py

```diff
@@ -279,3 +280,56 @@ def summarize_plant_to_location(df, grid):
     location_data = df.T.groupby(locations_in_df).sum().T
```

Comment:
Not part of this PR, but similar to `postreise.analyze.generation.summarize.get_generation_time_series_by_resources`, this can be simplified to `pg.groupby(locations_in_df, axis=1).sum()`.

Reply:
Done.